cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -54,24 +54,24 @@ var (
 		Buckets: prometheus.ExponentialBuckets(.1, 10, 7), // 100ms -> 27min
 	}, []string{"state"}) // state is an indexState
 
-	metricQueueLen = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "index_queue_len",
-		Help: "Current length of indexing queue by code host",
-	}, []string{"codehost"})
-
 	metricNumIndexed = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "index_num_indexed",
 		Help: "Number of indexed repos by code host",
 	}, []string{"codehost"})
 
-	metricFailedTotal = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "index_failed_total",
-		Help: "Counts failures to index",
+	metricNumNotIndexed = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "index_num_not_indexed",
```

Comment:
```suggestion
		Name: "index_num_unindexed",
```

I think this would be better wording